public/app/plugins/datasource/elasticsearch/datasource.ts

```diff
@@ -253,38 +280,54 @@ export class ElasticDatasource {
         continue;
       }
 
-      if (target.alias) {
-        target.alias = this.templateSrv.replace(target.alias, options.scopedVars, 'lucene');
-      }
-
       let queryString = this.templateSrv.replace(target.query, options.scopedVars, 'lucene');
       // Elasticsearch queryString should always be '*' if empty string
       if (!queryString || queryString === '') {
         queryString = '*';
       }
-      const queryObj = this.queryBuilder.build(target, adhocFilters, queryString);
+
+      let queryObj;
+      if (target.context && target.context === 'explore') {
```

Comment:
Think we can remove the `context` from the query now and only rely on `isLogsQuery` or does that screw up the check for empty query?